h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -151,6 +151,23 @@ public static void checkLeakedKeys() {
     _initial_keycnt = H2O.store_size();
   }
 
+  public static void checkArrays(double[] expected, double[] actual, double threshold) {
```

Comment:
Why not use `assertArrayEquals(double[], double[], double delta)` from JUnit ?

Reply:
Nope, I need to skip over NaNs and also, I need to change the threshold per element due to the fact that I want to scale the tolerance according to the element magnitudes.